### DISCUSSION TITLE
JDK 17 minimum required, so why use bcprov-jdk15to18 and bcpkix-jdk15to18?

### DISCUSSION THREAD

**Discussion by jimbogithub** [hashgraph/hedera-sdk-java#1787] [General]
`bcprov-jdk15to18` and `bcpkix-jdk15to18` are `for JDK 1.5 to JDK 1.8`.
The SDK states `JDK 17 is required` so why not use `bcprov-jdk18on` and `bcpkix-jdk18on`?  These are `for JDK 1.8 and up`.

**Comment by thenswan** [comment]
Thanks for pointing this out @jimbogithub. We're looking at our dependencies soon and these as well. If these dependencies are causing you any problems now, please let me know.
